defaults/host-ents.go

```diff
@@ -0,0 +1,197 @@
+	"github.com/docker/libentitlement/entitlement"
+	"github.com/docker/libentitlement/secprofile"
+	"github.com/docker/libentitlement/types"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"reflect"
+)
+
+const (
+	hostDomain = "host"
+
+	hostProcessesDomain = hostDomain + ".processes"
+	hostDevicesDomain   = hostDomain + ".devices"
+)
+
+const (
+	// HostDevicesNoneEntFullID is the ID for the host.devices.none entitlement
+	HostDevicesNoneEntFullID = hostDevicesDomain + ".none"
+	// HostDevicesViewEntFullID is the ID for the host.devices.view entitlement
+	HostDevicesViewEntFullID = hostDevicesDomain + ".view"
+	// HostDevicesAdminEntFullID is the ID for the host.devices.admin entitlement
+	HostDevicesAdminEntFullID = hostDevicesDomain + ".admin"
+
+	// HostProcessesNoneEntFullID is the ID for the host.processes.none entitlement
+	HostProcessesNoneEntFullID = hostProcessesDomain + ".none"
+	// HostProcessesAdminEntFullID is the ID for the host.processes.admin entitlement
+	HostProcessesAdminEntFullID = hostProcessesDomain + ".admin"
+)
+
+var (
+	hostDevicesNoneEntitlement  = entitlement.NewVoidEntitlement(HostDevicesNoneEntFullID, hostDevicesNoneEntitlementEnforce)
+	hostDevicesViewEntitlement  = entitlement.NewVoidEntitlement(HostDevicesViewEntFullID, hostDevicesViewEntitlementEnforce)
+	hostDevicesAdminEntitlement = entitlement.NewVoidEntitlement(HostDevicesAdminEntFullID, hostDevicesAdminEntitlementEnforce)
+
+	hostProcessesNoneEntitlement  = entitlement.NewVoidEntitlement(HostProcessesNoneEntFullID, hostProcessesNoneEntitlementEnforce)
+	hostProcessesAdminEntitlement = entitlement.NewVoidEntitlement(HostProcessesAdminEntFullID, hostProcessesAdminEntitlementEnforce)
+)
+
+var (
+	allowedMounts = []specs.Mount{
+		{
+			Destination: "/proc",
+			Type:        "proc",
+			Source:      "proc",
+			Options:     []string{"nosuid", "noexec", "nodev"},
+		},
+		{
+			Destination: "/dev",
+			Type:        "tmpfs",
+			Source:      "tmpfs",
+			Options:     []string{"nosuid", "strictatime", "mode=755"},
+		},
+		{
+			Destination: "/dev/pts",
+			Type:        "devpts",
+			Source:      "devpts",
+			Options:     []string{"nosuid", "noexec", "newinstance", "ptmxmode=0666", "mode=0620", "gid=5"},
+		},
+		{
+			Destination: "/sys",
+			Type:        "sysfs",
+			Source:      "sysfs",
+			Options:     []string{"nosuid", "noexec", "nodev", "ro"},
+		},
+		{
+			Destination: "/sys/fs/cgroup",
+			Type:        "cgroup",
+			Source:      "cgroup",
+			Options:     []string{"ro", "nosuid", "noexec", "nodev"},
+		},
+		{
+			Destination: "/dev/mqueue",
+			Type:        "mqueue",
+			Source:      "mqueue",
+			Options:     []string{"nosuid", "noexec", "nodev"},
+		},
+	}
+)
+
+func isAllowedMount(mount specs.Mount) bool {
+	for _, allowedMount := range allowedMounts {
+		if reflect.DeepEqual(mount, allowedMount) {
+			return true
+		}
+	}
+
+	return false
+}
+
+func hostDevicesNoneEntitlementEnforce(profile secprofile.Profile) (secprofile.Profile, error) {
```

Comment:
could you add a quick docstring to explain this entitlement and the other new ones?